app/export.go

```diff
@@ -334,3 +340,76 @@ func (a *App) BuildPostReactions(postId string) (*[]ReactionImportData, *model.A
 	return &reactionsOfPost, nil
 
 }
+
+func (a *App) ExportCustomEmoji(writer io.Writer, file string) *model.AppError {
+	customEmojiList, err := a.GetEmojiList(0, 100, model.EMOJI_SORT_BY_NAME)
```

Comment:
Why only 100 emojis? This whole function body should probably be in a loop doing 100 at a time until there are no more custom emoji left.